src/editor/data/previews/header-block.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import {a11yHiddenSvgProps} from '../../../utils/svg';
+
+const Header: React.FC<React.SVGProps<SVGSVGElement>> = (props) => (
```

Comment:
Should we named it more inforamtive? Like `HeaderIcon` ?

Reply:
I tried to name preview icons same as blocks.  All of them are in `./previews` folder, so I think it adds necessary context